### PR TITLE
vdk-frontend: include readmes for the data-pipelines folders

### PR DESCRIPTION
**Pull request by murphp15**
# Why
Before merging the frontend code we want to make sure the readmes are clearly explaining the project/where to find everything and how to buid everything. 

# What
1. Copied the readmes from the private repo. 
2. Removed references to CSP auth and replaced it with TODOs to be tackled once we can authenticate with a different oauth provider. 
3. removed references to extensibility.

### CODE REVIEW

File: projects/frontend/data-pipelines/projects/gui/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+is a subset of the E2E cypress tests that are filtered with `--env grepTags=@integration` and `--spec "e2e/integration/**/*.int.spec.js"`
+Example run: `npx cypress run --env OAUTH2_API_TOKEN=<the API token>,grepTags=@integration --spec "e2e/integration/**/*.int.spec.js"`
+
+_**Note**_: There are two levels of filtering of the E2E tests that constitutes the Integration Suite:
+1. Naming convention for the test filename: *.int.spec.js - to avoid unnecessary scanning and execution of before etc. hooks
+2. More granular tag selection: `grepTags=@integration`. e.g.: `it('Manage page - grid search', { tags: '@integration' }, () => {` or
+   `describe('Data Jobs Manage Page', { tags: '@integration' }, () => {`
+
+_**Important note**_: In order to ensure the ability to run the Integration Suite against the Data Jobs Wrapper UI and the Management Console UI
+as part of the TMS Staging pipeline, it is required to ensure that the test suite runs successfully against both the UIs(local and/or TMS Staging instance)
```
Comment (ivakoleva) [inline projects/frontend/data-pipelines/projects/gui/README.md:123]: what is TMS stagng

File: projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Taurus Data Pipelines
```
Comment (ivakoleva) [inline projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md:1]:
```suggestion
# Data Pipelines Frontend
```
or UI

File: projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+```
+
+### Include the module and router
+
+1. In `app.module.ts` include ngrx modules and actual Data Pipeline module
+```typescript
+imports: [
+  ...
+  VdkSharedCoreModule.forRoot(), // vdk shared core services
```
Comment (ivakoleva) [inline projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md:23]: Not planning ong renaming Taurus* class names, those names are already depended-on prior opening the source code; This is not a problem, there already are quite some packages and classes in the repo named after Taurus, it was the legacy name before open-sourcing.

Comment (murphp15) [reply projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md:23]: Ok cool!

File: projects/frontend/data-pipelines/projects/gui/README.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+The gui part of data-pipelines contains:
+
+  * **[Data Pipelines](projects/data-pipelines/README.md)** Library that holds UI Components for managing Data Pipeline API.
+  * **[UI](projects/ui/src)** Reference UI Wrapper Implementation of the Data Pipeline library
+
+The wrapper ([ui](projects/ui/src)) provides standard OAuth2 authorization flow, configuration of which could be found in [auth.ts](projects/ui/src/app/auth.ts)
+
+## Running and Testing
+
+You can use the reference implementation in `projects/ui` as showcase of the library.
```
[the remark this answers is not in the document]
Comment (murphp15) [reply projects/frontend/data-pipelines/projects/gui/README.md]: Teams within VMware which are using the screens in other projects are suppose to use `projects/ui` as a showcase of the library.
By removing that statement we are really taking focus away from extensibility and and how to test/view screens in the extensibility pattern. 
Let me know what you think?

File: projects/frontend/data-pipelines/projects/gui/README.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
+# VDK Data Pipeline UI
+
+The gui part of data-pipelines contains:
+
+  * **[Data Pipelines](projects/data-pipelines/README.md)** Library that holds UI Components for managing Data Pipeline API.
+  * **[UI](projects/ui/src)** Reference UI Wrapper Implementation of the Data Pipeline library
```
[the remark this answers is not in the document]
Comment (murphp15) [reply projects/frontend/data-pipelines/projects/gui/README.md:6]: Again if I remove reference we are taking focus away from the extensibility.

File: projects/frontend/data-pipelines/projects/gui/README.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
+# VDK Data Pipeline UI
+
+The gui part of data-pipelines contains:
+
+  * **[Data Pipelines](projects/data-pipelines/README.md)** Library that holds UI Components for managing Data Pipeline API.
```
[the remark this answers is not in the document]
Comment (murphp15) [reply projects/frontend/data-pipelines/projects/gui/README.md:5]: `projects/data-pipelines` is a library that holds UI Components. 
`projects/ui` along with internal projects in VMware consumes UI components.

So I defo think the wording here is correct.
 Please let me know if you disagree?

[the remark this answers is not in the document]
Comment (murphp15) [reply projects/frontend/data-pipelines/projects/gui/README.md:5]: done.

[the remark this answers is not in the document]
Comment (ivakoleva) [reply projects/frontend/data-pipelines/projects/gui/README.md:5]: what is "managing Data Pipeline API"

File: projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+```
+
+### Include the module and router
+
+1. In `app.module.ts` include ngrx modules and actual Data Pipeline module
+```typescript
+imports: [
+  ...
+  TaurusharedCoreModule.forRoot(), // taurus shared core services
```
Comment (ivakoleva) [inline projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md:23]: nit:
```suggestion
  TaurusharedCoreModule.forRoot(), // shared core services
```

File: projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+After building your library with `npm run build`, go to the dist folder `cd dist/data-pipelines` and run `npm publish`.
+
+### Running unit tests
+
+Run `npm run test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+### Testing
+
+You can use the reference implementation in <project_root>/projects/gui/projects/ui as showcase of the library.
```
Comment (ivakoleva) [inline projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md:81]:
```suggestion
You can use the implementation in <project_root>/projects/gui/projects/ui as showcase of the library.
```

File: projects/frontend/data-pipelines/projects/gui/README.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+
+### Running tests
+
+[Unit] Run `npm run test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+_**Note**_: Code coverage report will be generated in `reports/coverage`(for the UI Wrapper and the Data Pipeline Library)
+
+_**Note**_: Code coverage report for the Data Pipeline library will be also logged in the Console(and consumed by the CI/CD)
+
+TODO re-visit when OAuth is agreed on
```
Comment (ivakoleva) [inline projects/frontend/data-pipelines/projects/gui/README.md:89]: is there a ticket for this todo?

File: projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
+# Data Pipelines Frontend
+
+Data Pipelines help Data Engineers develop, deploy, run, and manage data processing workloads (called "Data Job").
```
Comment (ivakoleva) [inline projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md:3]: data pipelines, or the frontend of VDK?

**Review by ivakoleva — APPROVED** [review]

File: projects/frontend/data-pipelines/projects/gui/README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (murphp15) [reply projects/frontend/data-pipelines/projects/gui/README.md:5]: That is a copy and paste from the original repo. 
Would you prefer `interacting with Data Pipelines API`?

File: projects/frontend/data-pipelines/projects/gui/README.md (hunk 9 above)
Comment (murphp15) [reply projects/frontend/data-pipelines/projects/gui/README.md:89]: https://github.com/vmware/versatile-data-kit/issues/1610.
Updated comment also.

File: projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md (hunk 10 above)
Comment (murphp15) [reply projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md:3]: Fixed

**Review by gageorgiev — APPROVED** [review]

**Pull request merged**